docs/admin/config/batch_changes.mdx

```diff
@@ -344,12 +344,37 @@ However, if the user deletion is permanent, deleting both account and data, then
 
 ## Batch Spec Library
 
-<Callout type="note"> Batch Spec Library is currently in Experimental.</Callout>
+The Batch Spec Library accelerates batch change adoption by offering curated examples and guided templates that make large-scale code modifications accessible to developers at every skill level.
 
-The Batch Spec Library is a collection of Batch Specs that can be used to create Batch Changes. Sourcegraph provides a few Batch Specs out of the box.
+The library distinguishes between [templates](#templates) and [library examples](#library-examples), where library examples are complete batch specs and templates are library examples with variables.
+
+Sourcegraph instances come with a couple of library examples out of the box. You can use the GraphQL APIs to [manage the Batch Spec Library](#managing-the-batch-spec-library).
+
+As a site admin, you can [feature records](#featured-records) to highlight the most useful examples for your organization.
+
+### Library examples
+
+Library examples are complete batch specs intended for advanced users who are comfortable working with YAML and writing code. These examples serve as inspiration and starting points for custom batch changes, providing reference implementations that can be adapted to specific use cases.
+
+Library examples are visible in the library pane when you are in the batch spec editor. They are not displayed by default in the templates list which is the entry point for users creating a batch change. See [choosing a template](/batch-changes/create-a-batch-change#choosing-a-template) for more details.
```

Comment:
The end of this sentence "...which is the entry point for users creating a batch change." is a little confusing b/c it adds unnecessary info (IMO). Maybe just end this sentence at "...in the Template list."

Reply:
I added that sentence to give admins a reference to where the user guide would be. Do you think we should drop that?